api/src/pcapi/core/educational/api/stock.py

```diff
@@ -69,12 +69,10 @@ def edit_collective_stock(
     stock: educational_models.CollectiveStock, stock_data: dict
 ) -> educational_models.CollectiveStock:
 
-    beginning = stock_data.get("beginningDatetime")
-    beginning = serialization_utils.as_utc_without_timezone(beginning) if beginning else None
-    booking_limit_datetime = stock_data.get("bookingLimitDatetime")
-    booking_limit_datetime = (
-        serialization_utils.as_utc_without_timezone(booking_limit_datetime) if booking_limit_datetime else None
-    )
+    beginning_datetime = stock_data.get("beginningDatetime")
+    beginning = serialization_utils.as_utc_without_timezone(beginning_datetime) if beginning_datetime else None
+    booking_limit = stock_data.get("bookingLimitDatetime")
+    booking_limit_datetime = serialization_utils.as_utc_without_timezone(booking_limit) if booking_limit else None
```

Comment:
(style) Pourquoi avoir 2 variables distinctes pour chaque date ? Ça ouvre la porte à des bugs : le code plus bas pourrait utiliser `beginning_datetime` à la place de `beginning`. De plus, le renommage n'est pas homogène : on finit par utiliser la variable `beginning` (**sans** suffixe "_datetime") et la variable `booking_limit_datetime` (**avec** suffixe).


```suggestion
    beginning = stock_data.get("beginningDatetime")
    beginning = serialization_utils.as_utc_without_timezone(beginning) if beginning else None
    booking_limit = stock_data.get("bookingLimitDatetime")
    booking_limit = serialization_utils.as_utc_without_timezone(booking_limit) if booking_limit else None
```